core/src/impl/Kokkos_SharedAlloc.hpp

```diff
@@ -87,6 +87,8 @@ class SharedAllocationHeader {
         reinterpret_cast<char*>(alloc_ptr) - sizeof(SharedAllocationHeader));
   }
 
+  inline Record* get_record() { return m_record; }
```

Comment:
This is a member function that's defined inside of the class definition. `inline` is redundant.